grouper/graph.py

```diff
@@ -26,6 +26,14 @@
 def Graph():  # noqa
     return GroupGraph()
 
+# A GroupGraph caches permissions and groups as these objects which are intended
+# to behave like the corresponding models but without any connection to SQL
+# backend.
+PermissionTuple = namedtuple("CachedTuple",
```

Comment:
Yup.  Oversight on my part.  I had originally named them CachedTuples then quickly realized that that was far too vague and renamed them, but missed this.